### PR TITLE
SF-1205: Add fallback for `document.baseURI`

### PR DESCRIPTION
**Pull request by jrmykolyn**

### CODE REVIEW

File: packages/@storefront/core/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Fixed
+- SF-1205: Add fallback for document.baseURI.
```
Comment (Vic-Dev) [inline packages/@storefront/core/CHANGELOG.md]: Add something about support for ie

File: packages/@storefront/core/test/unit/services/urlUtils.ts [hunk 2]
```diff
@@ -16,6 +16,29 @@ suite('URL Service', ({ expect, spy, stub }) => {
...
+        win.document = {
+          querySelector: stub().withArgs('base').returns({ href: 'bar' }),
+        };
+
+        expect(Utils.getBaseUri()).to.eq('bar');
+      });
+
+      it('should fall back to the current location', () => {
+        win.location = { href: 'baz' };
+        win.document = { querySelector: spy() };
```
Comment (jmbtutor) [inline packages/@storefront/core/test/unit/services/urlUtils.ts]: This `spy` is not being used. You should probably have an assertion using it.

File: packages/@storefront/core/src/services/urlUtils.ts [hunk 3]
```diff
@@ -6,9 +6,13 @@ import UrlBeautifier from '../core/url-beautifier';
 import { WINDOW } from '../core/utils';
 
 namespace UrlUtils {
-  export const getBasePath = (): string => {
-    const basePath = UrlParse(WINDOW().document.baseURI).pathname;
+  export const getBaseUri = (): string  => {
+    const win: any = WINDOW();
```
Comment (Vic-Dev) [inline packages/@storefront/core/src/services/urlUtils.ts]: Do you have to cast this to `any`?

Comment (jrmykolyn) [reply packages/@storefront/core/src/services/urlUtils.ts]: Good point. Updated.

File: packages/@storefront/core/test/unit/services/urlUtils.ts (hunk 2 above)
Comment (jrmykolyn) [reply packages/@storefront/core/test/unit/services/urlUtils.ts]: Resolved.

File: packages/@storefront/core/CHANGELOG.md (hunk 1 above)
Comment (jrmykolyn) [reply packages/@storefront/core/CHANGELOG.md]: Updated.

**Comment by jmbtutor** [comment]
Demo passed.

**Comment by Vic-Dev** [comment]
LGTM! Don't forget to demo if you haven't already

**Comment by jrmykolyn** [comment]
> LGTM! Don't forget to demo if you haven't already

Awesome. Also, just demoed to @jmbtutor and @JoeyDeol on a Windows machine.

**Pull request merged**